CLI for Microsoft 365 v6.4

garrytrinder opened the pull request:
## Category

- [ ] Content fix
- [x] New article

## Contents of the Pull Request

Release announcement for the next release of CLI for Microsoft 365

Jwaegebaert reviewed (commented):
Awesome work with this release blog @garrytrinder! 💪 

I'm a big fan of the way you listed all the added commands under each heading for this release. I've left a few pointers that I noticed during my review that you can take a look at.

martinlingstuyl commented:
> Awesome work with this release blog @garrytrinder! 💪
> 
> I'm a big fan of the way you listed all the added commands under each heading for this release. I've left a few pointers that I noticed during my review that you can take a look at.

I think you're a bit too fast @Jwaegebaert, this is actually largely the content of the 6.3 blog post. @garrytrinder still needs to update this draft. 😅

Jwaegebaert commented:
wow, my bad 😅

martinlingstuyl commented:
But I'm glad you liked my 6.3 post 😏

Adam-it commented:
@garrytrinder I think we should add some small info about this change https://github.com/pnp/cli-microsoft365/pull/4460
due to this change we added additional permission to our app registration which is now needed for PP commands to continue working. So if some users have a script running using CLI after we swapped the resources the PP commands will stop working 🤔
In order to make them work again user needs to consent to the additional permission we now added so we should advise to do `m365 cli reconsent`

garrytrinder commented:
> @garrytrinder I think we should add some small info about this change [pnp/cli-microsoft365#4460](https://github.com/pnp/cli-microsoft365/pull/4460)
> due to this change we added additional permission to our app registration which is now needed for PP commands to continue working. So if some users have a script running using CLI after we swapped the resources the PP commands will stop working 🤔
> In order to make them work again user needs to consent to the additional permission we now added so we should advise to do `m365 cli reconsent`

Thanks for the heads-up @Adam-it, I've added it.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+  - CLI for Microsoft 365
+  - SharePoint
+  - SharePoint Framework (SPFx)
+type: regular
```
The type should be `popular` as this is the most recent announcement for the CLI. In place, the older blog op v6.3, the type there should replaced by `regular`

garrytrinder replied:
Ah in this case the v6.3 post was set to `regular` so that's why is used it here. I can replace the existing `popular` post though.

Jwaegebaert replied:
Then I think just updating this post to popular should suffice 😄

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+We've released three new commands in this release to manage the ownership of Power Automate Flows.
+
+To list the owners of a Power Automate Flow created by another user in by it's name, use:
```
To list the owners of a Power Automate Flow created by another user **by its** name, use:

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
````diff
@@ -0,0 +1,300 @@
+m365 flow owner remove --userName john.doe@contoso.com --environmentName Default-c5a5d746-3520-453f-8a69-780f8e44917e --flowName 72f2be4a-78c1-4220-a048-dbf557296a72
+```
+
+Also to adding specific users, we've also included support for adding a group as an owner.
````
Also to **add** specific users, **we've included** support for adding a group as an owner.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+### Retrieve tenant scoped SharePoint Online Application Customizers
+
+SharePoint Online Tenant Application Customizers are a type of SharePoint Framework (SPFx) extension. They allow developers to adding custom scripts or HTML to the header or footer of every page in a SharePoint site collection. A tenant scoped customization applies to all sites and subsites within the tenant.
```
SharePoint Online Tenant Application Customizers are a type of SharePoint Framework (SPFx) extension. They allow developers to **add** custom scripts or HTML to the header or footer of every page in a SharePoint site collection. A tenant scoped customization applies to all sites and subsites within the tenant.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+### Ensure a user on a SharePoint Online site
+
+When attempting to add a user to a SharePoint Online site that is already a member you will recieve an error.
```
When attempting to add a user to a SharePoint Online site that is already a member you will **receive** an error.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+### Manage SharePoint Online ListView Command Sets
+
+ListView Command Sets are a feature in SharePoint Online that allow you to add custom actions to the command bar in a list or library.
```
ListView Command Sets are a feature in SharePoint Online that **allows** you to add custom actions to the command bar in a list or library.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+We've added four new commands to this release to manage ListView Command Sets on SharePoint Online sites.
+
+To remove a ListView Command Set by its title on a specified SharePoint Onlin site, use:
```
To remove a ListView Command Set by its title on a specified SharePoint **Online** site, use:

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
````diff
@@ -0,0 +1,300 @@
+m365 spo commandset set --clientSideComponentId 799883f5-7962-4384-a10a-105adaec6ffc --newTitle 'Some new title' --location Both --webUrl https://contoso.sharepoint.com/sites/sales --scope Site
+```
+
+To retrieve a ListView Command Set by it's title from a SharePoint Online site, use:
````
To retrieve a ListView Command Set by **its** title from a SharePoint Online site, use:

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+If you have scripts that are using Power Platform commands and they stop working, it is highly likely that it will be due to this change. 
+
+To resolve the issue, you need to consent to the additional permission,PowerApps Service, that have been added to the PnP Management Shell Azure AD application.
```
To resolve the issue, you need to consent to the additional permission, **PowerApps** Service, that have been added to the PnP Management Shell Azure AD application.

garrytrinder replied:
Oh lol, so the service name is not the same as the product name 😂

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+### List Power Automate Flow runs for a flow created by another user
+
+Before the `flow run list` command only returned runs for Flows that were create by the logged in user. 
```
Before the `flow run list` command only returned runs for Flows that were **created** by the logged in user.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+### Turn on or off the Viva Connections start experience on a SharePoint Online Home Site
+
+In this release we have extended the `spo homeset set` command to now give the ability to turn on or off the Viva Connections start experience.
```
In this release we have extended the `spo homesite set` command to now give the ability to turn on or off the Viva Connections start experience.

Jwaegebaert commented on content/cli-for-microsoft-365/cli-for-microsoft-365-v6-4/index.md:
```diff
@@ -0,0 +1,300 @@
+
+We recently launched a community Discord server for CLI for Microsoft 365. 
+
+We've seen a huge amount of engagement and activity on the server. So much so, that we've decided to make it is where we want to focus and grow the CLI for Microsoft 365 community.
```
We've seen a huge amount of engagement and activity on the server. So much so, that we've decided to make it where we want to focus and grow the CLI for Microsoft 365 community.

Jwaegebaert reviewed (commented):
Awesome job @garrytrinder! I've left a few additional pointers 😄
